crates/stellar-registry-cli/README.md

```diff
@@ -66,17 +77,53 @@ stellar registry deploy \
 
 2. Install the deployed contract:
 ```bash
-stellar registry install my-token \
-  --out-dir ./contracts/my-token
+stellar registry install my-token
+```
+
+Then can interact with it the contract with `stellar-cli`:
+```bash
+stellar contract invoke --id my-token -- --help
 ```
 
 ## Registry Contract
 
-The registry contract is deployed on testnet at:
+You can install the registry contract like any other:
```

Comment:
What's the reason someone would want to do this? I'm unclear why this instruction is here

Reply:
This is just highlighting that the registry is just a normal contract and that you can interact with it directly if so desired. But mainly it was to highlight the `install` command on something that would work now. Since I still haven't published anything else to the contract yet...